modules/simulate-network-policies-ng20.adoc

```diff
@@ -17,8 +17,9 @@ The Network Graph does not display the generated policies in the visualization.
 . In the {product-title-short} portal, go to *Network Graph*.
 . Select a cluster, and then select one or more namespaces.
 . On the network graph header, select *Network policy generator*.
-. Optional: To generate a YAML file with network policies to use in the simulation, click *Generate and simulate network policies*. For more information, see "Generating network policies in the network graph".
-. Upload a YAML file of a network policy that you want to use in the simulation. The network graph view displays what your proposed network policies would achieve. Perform the following steps:
+. You can perform the following actions:
+* Generate a YAML file with network policies to use in the simulation by clicking *Generate and simulate network policies*. For more information, see "Generating network policies in the network graph".
+* Upload a YAML file of a network policy that you want to use in the simulation. Perform the following steps:
```

Comment:
Just wondering if it would be better to be more parallel with the above subbullet by saying

```suggestion
* Upload a YAML file of a network policy that you want to use in the simulation by performing the following steps:
```